package/base-files/files/bin/ipcalc.sh

```diff
@@ -32,17 +34,29 @@ function compl32(v,   ret) {
 	return ret
 }
 
+function notquad(addr) {
+	print "Not a dotted-quad "addr > "/dev/stderr"
+	exit(1)
+}
+
 BEGIN {
 	slpos=index(ARGV[1],"/")
 	if (slpos == 0) {
 		ipaddr=ip2int(ARGV[1])
+		if (ipaddr == "")
+			notquad(ARGV[1])
 		dotpos=index(ARGV[2],".")
 		if (dotpos == 0)
```

Comment:
The `else` has brackets, I would add them to the first branch as well (applying C coding style to AWK).